app/models/user.rb

```diff
@@ -6,64 +6,72 @@ class User < ActiveRecord::Base
 
   attr_accessible :email, :password, :password_confirmation, :remember_me,
-    :name, :image_url, :uploaded_image, :bio, :newsletter, :full_name,
-    :address_street, :address_number, :address_complement, :address_city,
-    :address_neighbourhood, :address_state, :address_zip_code, :phone_number,
-    :cpf, :state_inscription, :locale, :twitter, :facebook_link, :other_link,
-    :moip_login, :deactivated_at, :reactivate_token, :bank_account_attributes,
-    :access_type, :responsible_name, :responsible_cpf, :mobile_phone, :gender,
-    :staffs, :job_title, :birth_date, :admin, :original_doc1_url,
-    :original_doc2_url, :original_doc3_url, :original_doc4_url,
-    :original_doc5_url, :original_doc6_url, :original_doc7_url,
-    :original_doc8_url, :original_doc9_url, :original_doc10_url,
-    :original_doc11_url, :original_doc12_url, :original_doc13_url
+                  :name, :image_url, :uploaded_image, :bio, :newsletter, :full_name,
+                  :address_street, :address_number, :address_complement, :address_city,
+                  :address_neighbourhood, :address_state, :address_zip_code, :phone_number,
+                  :cpf, :state_inscription, :locale, :twitter, :facebook_link, :other_link,
+                  :moip_login, :deactivated_at, :reactivate_token, :bank_account_attributes,
+                  :access_type, :responsible_name, :responsible_cpf, :mobile_phone, :gender,
+                  :staffs, :job_title, :birth_date, :admin, :original_doc1_url,
+                  :original_doc2_url, :original_doc3_url, :original_doc4_url,
+                  :original_doc5_url, :original_doc6_url, :original_doc7_url,
+                  :original_doc8_url, :original_doc9_url, :original_doc10_url,
+                  :original_doc11_url, :original_doc12_url, :original_doc13_url
 
   enum access_type: [:individual, :legal_entity]
-  enum gender: [:male, :female]
+  enum gender:      [:male, :female]
 
   STAFFS = {
-    team: 0,
-    financial_board: 1,
-    technical_board: 2,
-    advice_board: 3
+    team:             0,
+    financial_board:  1,
+    technical_board:  2,
+    advice_board:     3
   }
 
   mount_uploader :uploaded_image, UserUploader
 
-  validates_length_of :bio, maximum: 140
-
-  validates_presence_of :email, :access_type
-  validates_uniqueness_of :email, allow_blank: true, if: :email_changed?, message: I18n.t('activerecord.errors.models.user.attributes.email.taken')
-  validates_format_of :email, with: Devise.email_regexp, allow_blank: true, if: :email_changed?
-
-  validates_presence_of :password, if: :password_required?
-  validates_confirmation_of :password, if: :password_confirmation_required?
-  validates_length_of :password, within: Devise.password_length, allow_blank: true
+  validates :bio, length: { maximum: 140 }
+  validates :access_type, presence: true
+  validates :email, presence: true
+  validates :email,
+              allow_blank: true,
+              uniqueness: true,
```

Comment:
Sim, por causa desse `if` aqui https://github.com/juntos-com-vc/juntos.com.vc/blob/609a0f5619b30e34c7af4b6e86576a9a8273007c/app/models/user.rb#L53. `allow_blank: true` só será realizada se a condição for satisfeita